### ISSUE TITLE
Help/Enhancement: Character limit

### ISSUE THREAD

**Issue by mm12** [brentadamson/dataconnector#46]
Every run I hit the 50,000 character limit, is there a way to fix it?

**Comment by brentadamson** [comment]
To clarify: is it the `Properties read/write` limit listed here? https://developers.google.com/apps-script/guides/services/quotas

**Comment by mm12** [comment]
Don't think so, those appear to be edit limits where this is a character limit. i think whatever its doing it too big to fit in a cell or something

**Comment by brentadamson** [comment]
Oh, I see. The returned data is too much for the cell to handle. We could probably truncate the response. Do you have a JMESPath filter applied?

**Comment by mm12** [comment]
Yes, it is a very large one too. Took a long time to make, I just have an extremely specific use where basically I need to whether or not about 90 values are present in an array.

**Comment by brentadamson** [comment]
I guess I'm still a little bit lost on why you need that many characters to fit into a cell. If you return an array of arrays in your JMESPath, you should be able to display the data across multiple cells. If the data isn't sensitive I can help in getting that setup if you need.
